sklego/preprocessing/pandastransformers.py

```diff
@@ -329,52 +344,68 @@ class ColumnSelector(BaseEstimator, TransformerMixin):
     import pandas as pd
     from sklego.preprocessing import ColumnSelector
 
-    df = pd.DataFrame({
+    df_pd = pd.DataFrame({
         "name": ["Swen", "Victor", "Alex"],
         "length": [1.82, 1.85, 1.80],
         "shoesize": [42, 44, 45]
     })
-    ColumnSelector(["length"]).fit_transform(df)
+
+    ColumnSelector(["length"]).fit_transform(df_pd)
     '''
         length
     0    1.82
     1    1.85
     2    1.80
     '''
 
-    # Selecting multiple columns from a pandas DataFrame
-    ColumnSelector(["length", "shoesize"]).fit_transform(df)
+    # Selecting multiple columns from a polars DataFrame
+    import polars as pl
+    from sklego.preprocessing import ColumnSelector
+
+    df_pl = pl.DataFrame({
+        "name": ["Swen", "Victor", "Alex"],
+        "length": [1.82, 1.85, 1.80],
+        "shoesize": [42, 44, 45]
+    })
+
+    ColumnSelector(["length", "shoesize"]).fit_transform(df_pl)
```

Comment:
what you show below here doesn't look like Polars output, I think that may need updating?

Reply:
Let me fix that

Reply:
![image](https://github.com/koaning/scikit-lego/assets/74981211/085b8bf3-5059-4550-81c7-762d2dc71aa3)
So I used polars output but it renders this way in mkdocs:

![image](https://github.com/koaning/scikit-lego/assets/74981211/886f993e-4915-4930-b3bd-d7be126e65b5)

Reply:
ah I think it's just a font that needs setting, the default one isn't exactly monospace

Reply:
Is that something I can fix or not really @FBruzzesi ?

Reply:
I never rendered that! @MarcoGorelli send help 😂!

Reply:
😄 dunno, in Narwhals I just have
```
theme:
  name: material
  font: false
```

Reply:
I recall doing stuff like that in the past as well, it's a monofont isnt always mono kind of a thing. Especially when you add emojis or fancy kinds of unicode to the mix.

Reply:
![image](https://github.com/koaning/scikit-lego/assets/74981211/3eff8e02-a21f-4164-bdc3-a27a4f25ca4b)
looks great now! I'll push it then